firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/RestaurantDetailFragment.kt

```diff
@@ -170,9 +170,7 @@ class RestaurantDetailFragment : Fragment(),
         // In a transaction, add the new rating and update the aggregate totals
         return firestore.runTransaction { transaction ->
             val restaurant = transaction.get(restaurantRef).toObject<Restaurant>()
-            if (restaurant == null) {
-                throw Exception("Resraurant not found at ${restaurantRef.path}")
-            }
+                ?: throw Exception("Resraurant not found at ${restaurantRef.path}")
```

Comment:
While we're at it, can we also fix this typo?

```suggestion
                ?: throw Exception("Restaurant not found at ${restaurantRef.path}")
```

Reply:
of course !